masschallenge/django-accelerator
# [AC-8490] Migrate `gender` data to `gender_identity`

## [AC-8490](https://masschallenge.atlassian.net/browse/AC-8490)

**Changes introduced**
- Add data migrations to migrate `gender` data to `gender_identity`

**Testing**
- Checkout to this branch and `$ make migrate`
- Confirm that all existing gender data is migrated to the gender_identity object according to the acceptance criteria on [AC-8490](https://masschallenge.atlassian.net/browse/AC-8490)
